lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -120,7 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $query->setMaxResults((int) $maxResult);
         }
 
-        if ($input->hasOption('show-sql')) {
+        if ($input->hasOption('show-sql') && $input->getOption('show-sql')) {
```

Comment:
In the current case, yes. But whenever for some reason the Option is missing (as someone might remove it, without removing this), this will never generate an exception.